src/benchmarks/micro/corefx/System.Linq/Perf.Linq.cs

```diff
@@ -230,7 +281,7 @@ public bool Contains_ElementNotFound(int size, int iterationCount, Perf_LinqTest
 
         [Benchmark]
         [ArgumentsSource(nameof(IterationSizeWrapperData))]
-        public bool Contains_FirstElementMatches(int size, int iterationCount, Perf_LinqTestBase.WrapperType wrapType)
+        public bool Contains_FirstElementMatches(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
```

Comment:
Changing parameter names breaks test identity in our reporting system - please revert all name changes.

Reply:
That is unfortunate, I have reverted the commit. Is the name `iteration` or `iterationCount` preferred for the new tests?

Reply:
consistency is the strong preference, so in this case I would use `iterationCount` for new tests on this type / in this file.

Reply:
Alright, that was already the case. Thanks!